src/System.Management.Automation/engine/hostifaces/PowerShell.cs

```diff
@@ -860,6 +861,25 @@ public static PowerShell Create(InitialSessionState initialSessionState)
             return result;
         }
 
+        /// <summary>
+        /// Constructs an empty PowerShell instance; a script or command must be added before invoking this instance.
```

Comment:
The comment should be more specific to this overload, like `Construct an empty PowerShell instance and associate it with the given Runspace`.

Reply:
Done.